modules/compute/gke-node-pool/variables.tf

```diff
@@ -386,6 +386,12 @@ variable "gke_version" {
   type        = string
 }
 
+variable "node_version" {
```

Comment:
Will remove this and use `gke_version` instead. Will make the changes tomorrow IST

Reply:
Node version is not required on prod clusters, removing this